spec/policies/user_policy_spec.rb

```diff
@@ -10,28 +10,37 @@
     permissions :show?, :update?, :edit?, :edit_intent?, :update_intent? do
       it { is_expected.to permit(user, user) }
     end
-    permissions :show?, :create?, :destroy?, :update?, :index?,
+    permissions :show?, :create?, :destroy?, :update?,
                 :edit?, :edit_intent?, :update_intent? do
```

Comment:
Should we move `:edit?` up to the previous line for consistency? Also in a few other places.